app/javascript/features/companies/manage/SquadBuilder.jsx

```diff
@@ -104,6 +112,12 @@ export const SquadBuilder = ({}) => {
 
   const isCompanyUnlocksChange = useSelector(selectIsCompanyUnlocksChanged)
 
+  const [openDelete, setOpenDelete] = React.useState(false)
+  const [openTransportDelete, setOpenTransportDelete] = React.useState(false)
+  
+  const [squad, setSquad] = React.useState(null)
+  const [transportUuid, setTransportUuid] = React.useState(null)
```

Comment:
Can we rename this to `transportOfSquadToDelete` to be more specific? Or similar name